src/ignitus-Shared/Components/InternshipList/index.ts

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import * as T from '../emotionStyles/shared'
+import * as C from '../emotionStyles/colors'
+
+export const InternshipTitle = styled(T.Title)`
+  color:orange
+`;
+
+export const SubHeading = styled(T.Title)`
+  font-size: 1.2em;
+`
+
+export const Description = styled(T.Paragraph)`
+	color: grey;
+	margin-bottom: 3px;
+`
+
+export const DateInfo = styled(T.Paragraph)`
+	color: grey;
+	right: 0;
+	position: absolute;
```

Comment:
Please don't use `position: absolute;  right: 0; top: 10px;` I don't understand their use-case they are generally helpful in layering content over each other like if we have to use `z-index` etc.